fr/_posts/2021-05-10-participation-approbateurs-conception.md

```diff
@@ -0,0 +1,96 @@
+---
+altLangPage: "2021/05/10/codesign-with-deciders"
+date: 2021-05-07
+description: " "
+title:  "Participation des approbateurs à la conception conjointe"

```

Comment:
```suggestion
title:  "Participation des décideurs à la conception conjointe"

```
Can you look into changing "approbateurs" to "décideurs" in French?

Reply:
Hum, les "approbateurs" de quelque chose, ce n'est pas la même chose que les "décideurs". De la même façon que "approvers" n'est pas la même chose que "decision makers". Approbateurs, ce n'est pas très commun, mais je crois que c'est le bon mot.

Reply:
D'accord.